src/Iframe.tsx

```diff
@@ -54,8 +54,10 @@ function Iframe(props: IframeProps) {
 
   // Reload on new revisions
   useEffect(() => {
-    if (reload?.revision) {
-      handleReload()
+    if (reload?.revision || reload.revision == 0) {
```

Comment:
@SimeonGriggs @thebiggianthead this crashes the studio if we dont have a revision property in our reload-object. Shouldn't this be `reload?.revision == 0`?

Reply:
@aulonm Thanks for flagging, my bad! Fix raised in [PR #8](https://github.com/SimeonGriggs/sanity-plugin-iframe-pane/pull/8).

Reply:
Fixed in `1.1.3`